flow360/component/simulation/validation/validation_utils.py

```diff
@@ -106,7 +106,7 @@ def check_deleted_surface_pair(value):
 
     # - Check if the surfaces are deleted.
     for surface in value.pair:
-        if surface._will_be_deleted_by_mesher(  # pylint:disable=protected-access
+        if isinstance(surface, Surface) and surface._will_be_deleted_by_mesher(  # pylint:disable=protected-access
```

Comment:
With @angranl-flex 's help, I fixed the issue updating the `check_deleted_surface_pair`